client-v2/src/shared/selectors/shared.ts

```diff
@@ -165,25 +168,31 @@ const articleFragmentIdSelector = (
   { articleFragmentId }: { articleFragmentId: string }
 ) => articleFragmentId;
 
-const supportingArticlesSelector = createSelector(
-  articleFragmentsFromRootStateSelector,
-  articleFragmentIdSelector,
-  (articleFragments, id) =>
-    (articleFragments[id].meta.supporting
-      ? articleFragments[id].meta.supporting!
-      : []
-    ).map((sId: string) => articleFragments[sId])
-);
+const createSupportingArticlesSelector = () =>
+  createSelector(
+    articleFragmentsFromRootStateSelector,
+    articleFragmentIdSelector,
+    (articleFragments, id) =>
+      (articleFragments[id].meta.supporting
+        ? articleFragments[id].meta.supporting!
+        : []
+      ).map((sId: string) => articleFragments[sId])
+  );
 
```

Comment:
Is there a reason we're exporting both selectors and selector factories here? It seems likely a developer would use the selector rather than the factory by mistake, and miss out on yummy memoisation goodness! Or is is that there are currently components using the selectors, and that we need to refactor further down the line?